typescript/vrotsc/src/compiler/transformer/fileTransformers/workflow/decorators.ts

```diff
@@ -51,8 +52,12 @@ export function registerMethodDecorators(methodNode: ts.MethodDeclaration, itemI
 	}
 
 	for (const decoratorNode of decorators) {
-		const itemStrategy = getItemStrategy(decoratorNode, itemInfo);
-		itemStrategy.registerItemArguments(decoratorNode);
+		const itemStrategy = getItemStrategy(decoratorNode);
+		if (itemStrategy.getDecoratorType() !== WorkflowItemType.RootItem) {
```

Comment:
Just a side note: I think it would be good to introduce a new ` WorkflowItemType` called `EndItem` and this end item to be able to be linked for example to a decision item (as an alternative exit).

Reply:
That's a todo for later for sure